bin/discovery.rb

```
@@ -188,6 +189,14 @@ def protection_domain_sdslist(protection_domain)
   transport.post_request(url, {}, "get") || []
 end
 
+def protection_domain_sdrlist(protection_domain)
+  sp_url = "/api/instances/ProtectionDomain::%s/relationships/Sdr" % [protection_domain["id"]]
+  url = transport.get_url(sp_url)
+  transport.post_request(url, {}, "get") || []
+rescue
+  []
```

Maybe a `logger.warn` would be good here? Is there a known case where it blows up? I'd rather just eat the errors in the known case and either raise or at least log the unknown  cases.

---

@gavin-scott yes can add a logger warn... any version of flex os earlier than 3.5, I realized too I should have called is sdr_url